test/resource/changes/cascade_update_test.exs

```diff
@@ -0,0 +1,292 @@
+defmodule Ash.Test.Resource.Change.CascadeUpdate do
+  @moduledoc false
+  use ExUnit.Case, async: true
+
+  alias Ash.Test.Domain
+  alias Ash.Test.Resource.Change.CascadeUpdate, as: Test
+
+  defmodule Notifier do
+    @moduledoc false
+    use Ash.Notifier
+
+    def notify(notification) do
+      if notification.action.name == :update do
+        Agent.update(
+          Test.Agent,
+          &%{&1 | notifications: MapSet.put(&1.notifications, notification.data.id)}
+        )
+      end
+
+      :ok
+    end
+  end
+
+  defmodule Author do
+    @moduledoc false
+    use Ash.Resource, domain: Domain, data_layer: Ash.DataLayer.Ets
+
+    attributes do
+      uuid_primary_key :id
+      attribute :name, :string, public?: true
+    end
+
+    actions do
+      default_accept :*
+      defaults [:read, create: :*]
+
+      update :update do
+        primary? true
+        require_atomic? false
+        skip_unknown_inputs [:content]
+
+        change cascade_update(:posts,
+                 copy_inputs: [:name, :content],
+                 return_notifications?: true
+               )
+      end
+
+      update :update_atomic do
+        change cascade_update(:posts,
+                 copy_inputs: [:name]
+               )
+      end
+
+      update :wrong_relationship_cascade do
+        change cascade_update(:postssss,
+                 copy_inputs: [:name]
+               )
+      end
+
+      update :no_notification_update do
+        change cascade_update(:posts,
+                 return_notifications?: true,
+                 action: :no_notification_update
+               )
+      end
+    end
+
+    relationships do
+      has_many :posts, Test.Post, public?: true
+    end
+
+    code_interface do
+      define :create
+      define :update
+      define :no_notification_update
+      define :read
+    end
+  end
+
+  defmodule Post do
+    @moduledoc false
+    use Ash.Resource, domain: Domain, data_layer: Ash.DataLayer.Ets, notifiers: [Test.Notifier]
+
+    attributes do
+      uuid_primary_key :id
+      attribute :name, :string, public?: true
+    end
+
+    actions do
+      default_accept :*
+      defaults [:read, create: :*]
+
+      update :update do
+        primary? true
+        require_atomic? false
+        skip_unknown_inputs [:content]
```

Comment:
Since :content is downstream, I have to skip it here, which then removes it from changes in the changeset.